backend/addcorpus/json_corpora/export_json.py

```diff
@@ -0,0 +1,95 @@
+from typing import Dict
+from datetime import date
+from addcorpus.models import Corpus, CorpusConfiguration, Field
+from addcorpus.json_corpora.constants import DEFAULT_CSV_DELIMITER, DATE_FORMAT
+from addcorpus.es_mappings import primary_mapping_type
+
+def export_json_corpus(corpus: Corpus) -> Dict:
+    config = corpus.configuration
+    data = {'name': corpus.name}
+    data['meta'] = export_corpus_meta(config)
+    data['source_data'] = export_corpus_source_data(config)
+    options = export_corpus_options(config)
+    if options:
+        data['options'] = options
+    data['fields'] = [
+        export_json_field(field) for field in config.fields.all()
+    ]
+    return data
+
+def export_corpus_meta(configuration: CorpusConfiguration) -> Dict:
+    return {
+        'title': configuration.title,
+        'category': configuration.category,
+        'description': configuration.description,
+        'languages': configuration.languages,
+        'date_range': {
+            'min': export_date(configuration.min_date),
+            'max': export_date(configuration.max_date),
+        }
+    }
+
+def export_date(date: date):
+    return date.strftime(DATE_FORMAT)
+
+def export_corpus_source_data(configuration: CorpusConfiguration) -> Dict:
+    data = {
+        'type': 'csv'
+    }
+    if configuration.source_data_delimiter != DEFAULT_CSV_DELIMITER:
```

Comment:
I believe `configuration.source_data_delimiter` has a default value right? In that case the check can be skipped and just set to that value.

Reply:
Yes, but if you are using the default value, you don't need to specify the delimiter in the JSON at all. So this check is needed if importing + exporting should return the same JSON.